apb-base/Dockerfile

```diff
@@ -1,41 +1,38 @@
-# Pastebin wNTPjTND
 FROM centos:7
-# MAINTAINER {{ $MAINTAINER }}
+MAINTAINER Ansible Playbook Bundle Community
 
 LABEL "com.redhat.apb.version"="0.1.0"
 
-RUN mkdir -p /root/.kube
+RUN mkdir -p /root/.kube /usr/share/ansible/openshift \
+            /etc/ansible /opt/apb /opt/ansible
 COPY config /root/.kube/config
-RUN yum -y update \
- && yum -y install epel-release centos-release-openshift-origin \
- && yum -y install origin origin-clients net-tools bind-utils \
- && yum install -y python-setuptools python-pip gcc python-devel python-cffi openssl-devel \
- && yum clean all
+RUN yum -y install epel-release centos-release-openshift-origin \
+    && yum -y update \
+    && yum -y install origin origin-clients net-tools bind-utils \
+    && yum install -y python-setuptools python-pip gcc python-devel python-cffi openssl-devel \
+    && yum clean all
 
 RUN git clone https://github.com:/ansible/ansible.git
 RUN cd ansible \
- && pip install -U setuptools \
- && git checkout devel \
- && python setup.py install
+    && pip install -U setuptools \
+    && git checkout devel \
+    && python setup.py install
 
 RUN git clone https://github.com/openshift/openshift-restclient-python
 RUN cd openshift-restclient-python \
- && pip install -r requirements.txt \
- && python setup.py develop \
- && source ../ansible/hacking/env-setup
-RUN openshift-ansible-gen modules
-
-RUN mkdir -p /usr/share/ansible/openshift
-RUN mkdir /etc/ansible
-RUN cp _modules/*.py /usr/share/ansible/openshift
-RUN echo "localhost ansible_connection=local" > /etc/ansible/hosts
-RUN echo '[defaults]' > /etc/ansible/ansible.cfg
-RUN echo 'roles_path = /opt/ansible/roles' >> /etc/ansible/ansible.cfg
-RUN echo 'library = /usr/share/ansible/openshift' >> /etc/ansible/ansible.cfg
-
-RUN mkdir /opt/apb
-
-COPY oc-login.sh /usr/bin
-COPY entrypoint.sh /usr/bin
+    && pip install -r requirements.txt \
+    && python setup.py develop \
+    && source ../ansible/hacking/env-setup
+RUN openshift-ansible-gen modules -o /usr/share/ansible/openshift
+
+RUN echo "localhost ansible_connection=local" > /etc/ansible/hosts \
+    && echo '[defaults]' > /etc/ansible/ansible.cfg \
+    && echo 'roles_path = /opt/ansible/roles' >> /etc/ansible/ansible.cfg \
+    && echo 'library = /usr/share/ansible/openshift' >> /etc/ansible/ansible.cfg
+
+COPY oc-login.sh entrypoint.sh /usr/bin/
+
+RUN useradd -u 1001 -r -g 0 -M -b /opt/apb -s /sbin/nologin -c "apb user" apb
```

Comment:
Also just want to make sure that this won't break when the same useradd command is run in one of the Dockerfiles based on this one. If that will break it, would you mind updating the other Dockerfiles (postgres and postgres-demo were just added) for the sample apps to remove this? If not then :shipit: